src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/elevator/ElevatorPlate.java

```diff
@@ -96,7 +97,10 @@ public BlockUseHandler getItemHandler() {
         LinkedList<ElevatorFloor> floors = new LinkedList<>();
         int index = 0;
 
-        for (int y = 0; y < b.getWorld().getMaxHeight(); y++) {
+        int minY = SlimefunPlugin.getMinecraftVersion().isAtLeast(MinecraftVersion.MINECRAFT_1_16)
+            ? b.getWorld().getMinHeight() : 0;
```

Comment:
We should have a helper method for that, like a `WorldUtils` class for these things